package.json

```diff
@@ -16,6 +15,14 @@
     "process-readme": "markdown-toc -i README.md && prettier --write README.md && git add README.md",
     "dist": "./scripts/dist.sh"
   },
+  "keywords": [
+    "csaf",
+    "csaf-validator-service",
```

Comment:
Please add the following keywords:
- "csaf full validator"
- "csaf extended validator"
- "csaf basic validator"